src/index.css

```diff
@@ -10,6 +10,138 @@
   max-height: 660px;
 }
 
+div.cio-autocomplete .cio-submit-btn, .cio-clear-btn {
```

Comment:
How do we feel about the specificity of these?

Reply:
I was using the default styles at first and I needed the element selectors to make it specific. Now that I'm not using the default styles, I think they are unnecessary. I'll update